lib/repl.js

```diff
@@ -1345,6 +1334,15 @@ function isCodeRecoverable(code) {
   return stringLiteral ? lastChar === '\\' : isBlockComment;
 }
 
+function _parseREPLKeyword(repl, keyword, rest) {
```

Comment:
Perhaps rather than passing `repl` in as an argument, restructure this to assume `this` === `repl` ... such that in the `util.deprecate` call above you can do:

```js
util.deprecate(_parseREPLKeyword, '...', '...');
```